From fab65471835c3486ffd7074735b8283e95de22cb Mon Sep 17 00:00:00 2001
From: Matt Selsky <matthew.selsky@twosigma.com>
Date: Tue, 29 Nov 2016 19:59:07 -0500
Subject: [PATCH 075/268] Rename loop variable to fix shadowing of class name

pyflakes reported:
ntpq/ntpq:1642: import 'cmd' from line 14 shadowed by loop variable
---
 ntpq/ntpq | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ntpq/ntpq b/ntpq/ntpq
index 54c93f7..91a607a 100755
--- a/ntpq/ntpq
+++ b/ntpq/ntpq
@@ -1637,8 +1637,8 @@ if __name__ == '__main__':
         else:
             for ihost in interpreter.chosts:
                 if session.openhost(*ihost):
-                    for cmd in interpreter.ccmds:
-                        interpreter.onecmd(cmd)
+                    for command in interpreter.ccmds:
+                        interpreter.onecmd(command)
                     session.close()
         raise SystemExit(0)
     except KeyboardInterrupt:
-- 
2.7.4

